speech/api/speechrest.py

```diff
@@ -29,7 +29,7 @@
 
 # Path to local discovery file
 # [START discovery_doc]
-API_DISCOVERY_FILE = 'speech-discovery_google_rest_v1.json'
+API_DISCOVERY_FILE = __file__.rsplit('/', 1)[0] + '/speech-discovery_google_rest_v1.json'
```

Comment:
Yeah - I originally had that, but then I had to add an import that felt irrelevant to the sample.
*shrug. I guess it's good to do it the Right Way...